src/test/resources/org/broadinstitute/hellbender/tools/mutect/mito/unfiltered.vcf

```diff
@@ -34,9 +34,9 @@
 ##filtering_status=Warning: unfiltered Mutect2 calls.  Please run FilterMutectCalls to remove false positives.
 ##source=Mutect2
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878
-chrM	152	.	T	C	.	.	DP=1582;ECNT=1;TLOD=5266.19;POPAF=5.000e-08	GT:AD:AF:F1R2:F2R1:MBQ:MFRL:MMQ:MPOS:OCM:POTENTIAL_POLYMORPHIC_NUMT	0/1:3,1556:0.998:2,777:1,779:30,30:16270,369:60,60:42:0:true
-chrM	263	.	A	G	.	.	DP=858;ECNT=4;TLOD=2641.72;POPAF=5.000e-08	GT:AD:AF:F1R2:F2R1:MBQ:MFRL:MMQ:MPOS:OCM	0/1:1,831:0.999:0,403:1,428:10,30:292,305:60,60:32:800
```

Comment:
Did you edit this by hand? For the second variant I think 800 should have been deleted instead of 32.

For the record, I'm not wasting time going through every vcf file looking for typos - I just happened to notice this particular one (and it could very well be not a typo)

Reply:
You're right.  Fixed.